### ISSUE TITLE
Disable Google Play message

### ISSUE THREAD

**Issue by disanvparean** [aminecmi/ReaderForSelfoss-tracker#24]
Hello,
Firstly, thank you for you app, it is very good.
But, can you please allow me to disable really annoying Google Play message that says that I don't have the application from the play store so the developer is not responsible if it is a fake.
I don't use Google Play Store for privacy reasons, I use Yalp Store which download from the Play Store.
Thank you, and keep the good work :)

**Comment by aminecmi** [comment]
Hi,

I won't delete the message. Not completely anyway.

It's there because the APK of my app is on multiple websites, and I can't assure the user that the APK is safe from any tampering.
The only, truly, safe version is on the play store. 

What I could do is provide an APK on this repository, that wouldn't have the modal. Users using an APK from outside the play store or my repository, would keep seeing the modal, but with a link to the genuine APK. I may also decrease the amount of time the modal is displayed.

Providing the APK would also mean that I'll have to , and providing a "check for update" functionality.

What do you think about these changes ?

**Comment by disanvparean** [comment]
Hello,
Thank you for your answer.
I completely understand why you don't want to delete it, and I think it is a good thing.
I would like to have an option allowing me to disable it. It is displayed every time I use the application, every time I click on a link and then come back to the application.
I don't think it is necessary to bother you to create a repository for the app.

**Comment by aminecmi** [comment]
I'm not comfortable with adding the option. 
That would mean that any user could disable it without thinking, and then, get mad about their version of the app causing issues.

Still, I totally understand the problem with the message being displayed that much.

Providing the apk will not be a burden, and it can be done easily. It will be available tomorrow at the latest tomorrow.

**Comment by aminecmi** [comment]
I'll try to handle this as fast as I can.

**Comment by aminecmi** [comment]
@disanvparean The apk will be available in the release page of this repository.
I will deploy it tomorrow when all the translations are available.

**Comment by aminecmi** [comment]
@disanvparean The apk is available on the release page [here](https://github.com/aminecmi/ReaderForSelfoss-tracker/releases/tag/1.4.0.4).

There will be update messages when a new version is available (that you can ignore and will be displayed only once by update).

If you find any other issue with the app, don't hesitate to open a new issue.

**Comment by disanvparean** [comment]
Wow, I'm impressed.
Thank you very much :+1:
